packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/users/resetPassword/ResetPassword.modal.tsx

```diff
@@ -80,22 +85,65 @@ const ResetUserPassword = () => {
             {t('resetUserPasswordTitle')}
           </DialogTitle>
           {newPass ? (
-            <Alert variant="success">
+            <div data-testid="pwd-connection-info">
               <p>{t('resetUserPasswordSuccess')}</p>
-              <div className="relative my-4">
-                <Button
-                  onClick={() => handleCopyPass()}
-                  className="absolute top-0 right-0 m-2 p-2 text-sm bg-primary-500 text-white rounded hover:bg-primary-700 transition duration-300"
-                  data-testid="reset-password-copy-button"
-                >
-                  <Copy className="size-4" />
-                  <span className="sr-only">copy</span>
-                </Button>
-                <pre className="p-4 bg-gray-100 rounded">
-                  <code>{newPass}</code>
-                </pre>
+              <div data-testid="code-pwd-container" className="p-2">
+                <Code
+                  code={newPass}
+                  label={t('resetUserPasswordCode')}
+                  lang={docker}
+                  theme={githubDark}
+                  onCopied={() =>
+                    toast.toast({
+                      title: t('resetUserPasswordCopy'),
+                    })
+                  }
+                />
+              </div>
+              <p>{t('resetUserConnectionTitle')}</p>
+              <div data-testid="code-uri-container" className="p-2">
+                {service.endpoints.length > 1 && (
+                  <Select
+                    value={selectedEndpoint.component}
```

Comment:
So the `selectedEndpoint` is possibly `undefined` :)
```suggestion
                    value={selectedEndpoint?.component}
```

Reply:
Hello, yes you are right, fixed :)